app/javascript/admin/adminModal.js

```diff
@@ -17,7 +17,7 @@ const adminModal = (
   cancelBtnText,
   cancelBtnAction,
 ) => `
-  <div class="crayons-modal crayons-modal--s absolute">
+  <div class="crayons-modal crayons-modal--s fixed">
```

Comment:
@ludwiczakpawel This is a sidebar conversation but it came up with this PR. I noticed that the only way for our `crayons-modal` to render successfully is if it uses a `fixed` position. Could we explore adding this position to the `crayons-modal` itself? Or are there scenarios where we wouldn't want this to happen?

Reply:
Can you give me a bit more context and/or maybe a demo? `.crayons-modal` [element already have `position: fixed` applied from SCSS file level](https://github.com/forem/forem/blob/master/app/assets/stylesheets/components/modals.scss#L26).

Why would you need to change it and what are you trying to achieve here? Maybe I could help.

Reply:
Ah okay, I didn't realize that `fixed` is *already* apart of `crayons-modal`. I think the addition of the `fixed` utility class is being misused; let me remove it, and that should solve everything.

Thank you!